spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET #index' do
+    before(:example) { get '/users/:user_id/posts' }
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "renders 'index' template" do
+      expect(response).to render_template('index')
+    end
+
+    it 'has a text in the view' do
+      expect(response.body).to include('Find me in app/views/posts/index.html.erb')
+    end
+  end
+
+  describe 'GET #show' do
+    before(:example) { get '/users/:user_id/posts/show' }
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "renders 'show' template" do
+      expect(response).to render_template('show')
+    end
+
+    it 'has a text in the view' do
+      expect(response.body).to include('Find me in app/views/posts/show.html.erb')
+    end
+  end
+end
```

Comment:
- [ ] Please fix your test currently are not passing
![test](https://user-images.githubusercontent.com/44745855/144416443-6e94e9d2-eef7-43b5-b10d-4b44a5a18bf3.png)

Reply:
Hi @DeliceLydia, can you show me the description of the failing tests? because the test works well in my machine.

![image](https://user-images.githubusercontent.com/77190203/144427231-91697344-9c3a-44fc-a127-ffa645bee5d7.png)

Reply:
All tests are running and passing but for that, I need to install the `npm` packages.